src/webvr-polyfill.js

```diff
@@ -100,7 +100,7 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
   navigator.getVRDisplays = this.getVRDisplays.bind(this);
 
   // Polyfill native VRDisplay.getFrameData
-  if (this.nativeWebVRAvailable && this.isCardboardCompatible() && window.VRFrameData) {
+  if (this.nativeWebVRAvailable && window.VRFrameData) {
```

Comment:
I think there should be a check for `if (window.VRDisplay.prototype.getFrameData) { … }` [here](https://github.com/tbalouet/webvr-polyfill/blob/1ac010e/src/webvr-polyfill.js#L251)